templates/beta-release-post.adoc

```diff
@@ -103,21 +103,47 @@ To enable the new beta features in your app, add them to your `server.xml`:
 [#run]
 === Try it now 
 
-To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE XX, Java SE 17, Java SE 11, and Java SE 8.
+To try out these features, update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE XX, Java SE 17, Java SE 11, and Java SE 8.
 // // // // // // // //
 // In the preceding section:
 // Check if a new non-LTS Java SE version is supported that needs to be added to the list (17, 11, and 8 are LTS and will remain for a while)
+// https://openliberty.io/docs/latest/java-se.html
 // // // // // // // //
 
-If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], you can install the All Beta Features package using:
 
+[source,xml]
+----
+<plugin>
+    <groupId>io.openliberty.tools</groupId>
+    <artifactId>liberty-maven-plugin</artifactId>
+    <version>3.3.4</version>
+    <configuration>
+        <runtimeArtifact>
+          <groupId>io.openliberty.beta</groupId>
+          <artifactId>openliberty-runtime</artifactId>
+          <version>RELEASE_VERSION</version>
```

Comment:
I think the `<version>RELEASE_VERSION</version` could make it too easy for someone to replace with "22.0.0.12" which wouldn't work.

We could do: 
  `<version>YY.0.0.MM-beta</version>`

or even just  
 `<version>LATEST</version>`

Reply:
This a template file for the beta release blog post.  There's a GH Action that replaces RELEASE_VERSION throughout the file with the passed in release, like 23.0.0.1-beta